### DISCUSSION TITLE
Do docs contributions like typo fixes really need an issue as CONTRIBUTING.md says?

### DISCUSSION THREAD

**Discussion by corneliusroemer** [keycloak/keycloak#35211] [Ideas]
I've recently opened two PRs for typos in the documentation:
- https://github.com/keycloak/keycloak/pull/35210
- https://github.com/keycloak/keycloak/pull/35163

Per CONTRIBUTING.md, you want issues for each PR. Does that apply even to such docs typos? That would essentially double the work for contributors and add to the pile of issues (1.7k right now). But if that's an absolute requirement, sure, I'll do it. It'd be nice if CONTRIBUTING.md stated exceptions from the issues rule (if there are any).

https://github.com/keycloak/keycloak/blob/05116f7951a9e1413c0a90c24b4648ee4a81ef4d/CONTRIBUTING.md?plain=1#L19

**Comment by jonkoops** [comment]
Yes, they do. We need this for tracking purposes so the changelog is properly populated, as well as a tracker for any possible backports.

**Reply by corneliusroemer** [reply]
Would be great to add this to CONTRIBUTING.md, it's much easier to accept seemingly pointless process if there's a good reason for it (and the process isn't pointless after all!).

**Reply by corneliusroemer** [reply]
I've made an issue proposing this addition :) https://github.com/keycloak/keycloak/issues/35252

**Reply by jonkoops** [reply]
Seems like a fair assessment, feel free to raise a PR for it.
